### PR TITLE
Wali/support php 8.2.6 and 8.1.19   1831694

### PR DESCRIPTION
**Pull request by waliMSFT**
- updated php versions: 8.2.6 and 8.1.19 (partners asked for these versions)
- updated php versions: 8.2.7 and 8.1.20 (latest versions)

### CODE REVIEW

File: images/runtime/php-fpm/8.1/base.bullseye.Dockerfile [hunk 1]
```diff
@@ -0,0 +1,137 @@
+FROM oryxdevmcr.azurecr.io/private/oryx/php-fpm-8.1
```
Comment (pauld-msft) [inline images/runtime/php-fpm/8.1/base.bullseye.Dockerfile:1]: These should be gitignored now, so we can safely remove these `base.*.Dockerfiles` from the git index with `git rm --cached <file>`

File: images/runtime/php/8.2/base.bullseye.Dockerfile [hunk 2]
```diff
@@ -1,148 +0,0 @@
-FROM oryxdevmcr.azurecr.io/private/oryx/php-8.2
```
Comment (snehapar9) [inline images/runtime/php/8.2/base.bullseye.Dockerfile:1]: Curious why we are deleting base.*.Dockerfile?

**Review by snehapar9 — APPROVED** [review]
LGTM. Approved with one clarification request.

File: images/runtime/php/8.2/base.bullseye.Dockerfile (hunk 2 above)
Comment (cormacpayne) [reply images/runtime/php/8.2/base.bullseye.Dockerfile:1]: The comment from @pauld-msft above led to this removal: we previously added these auto-generated Dockerfiles to the `.gitignore`, but if the files were already a part of the repository when the changes to the `.gitignore` were made, Git doesn't remove the files. The command Paul mentioned will remove files that are already a part of the repository that match the `.gitignore` changes.

**Review by cormacpayne — APPROVED** [review]
LGTM :shipit:

**Comment by harryli0108** [comment]
Just one quick question. Why did we stopped using the `base.bullseye.Dockerfile`?
Other than that, all looks good to me as well. Thanks Wali!

File: images/runtime/php/8.2/base.bullseye.Dockerfile (hunk 2 above)
Comment (snehapar9) [reply images/runtime/php/8.2/base.bullseye.Dockerfile:1]: Thanks for clarifying @cormacpayne!

**Review by william-msft — APPROVED** [review]

**Pull request merged**